### PR TITLE
chore: housekeeping

### PR DESCRIPTION
**Pull request by barmac**
* update dependencies
* replace SVG loader/plugin with actively maintained @svgr
* migrate to webpack@5 and rollup@4

Fixes #

### Proposed Changes

*
*
*

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "properties-panel"
   ],
   "main": "dist/index.js",
-  "module": "dist/index.esm.js",
```
Comment (nikku) [inline package.json:34]: Last time we tried we found this causes issues. Did our assessment change? 

I'd at least make this explicit via a separate commit.

Comment (barmac) [reply package.json:34]: Hmm I thought it worked fine. I'll revert this change as I'm not 100% sure, and we have not adopted it in bpmn-js-properties-panel yet.

Comment (nikku) [reply package.json]: If I remember it failed with `feel-editor` or something for older bundlers.

**Review by nikku — APPROVED** [review]
